torch/lib/TensorLib/HalfConvert.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
```

Comment:
We don't want to expose THC (CUDA) or TH half types at the public API level, so we need to write our own.